src/notify.js

```diff
@@ -35,7 +36,6 @@ class Toast extends React.Component {
 
 		const containerStyle = {
 			position: 'fixed',
-			width: '50%',
```

Comment:
What's the behaviour when this width is removed?. (I agree that 50% is not a very smart default here). But does it break layout if you don't pass the fullWidth setting?